web/blog/2022-11-29-typescript-feature-announcement.md

```diff
@@ -56,7 +56,7 @@ This will generate a project skeleton in the folder `ts-project`. The project st
 ```
 
 
-At this point, we can chose one of three options:
+At this point, we can choose one of three options:
```

Comment:
I fixed some of the other files I touched since I have Grammarly installed in my VS Code.